agreement/service_test.go

```diff
@@ -583,51 +583,82 @@ func (l amCoserviceListener) dec(sum uint) {
 	}
 }
 
+// copied from fuzzer/ledger_test.go. We can merge once a refactor seems necessary.
+func generatePseudoRandomVRF(keynum int) *crypto.VRFSecrets {
+	seed := [32]byte{}
+	seed[0] = byte(keynum % 255)
+	seed[1] = byte(keynum / 255)
```

Comment:
Alternatively, `binary.PutVarint(seed[:], keynum)`.

https://golang.org/pkg/encoding/binary/

Reply:
Works too. (Though would prefer not to import another pkg, will merge barring objections)